core_lib_generator/template_core_lib/core_lib/config/template_core_lib.yaml

```diff
@@ -0,0 +1,16 @@
+# @package _global_
+core_lib:

```

Comment:
how come this file has content? are we generating the config completely?

Reply:
changed the contents to template